cms/templates/admin/cms/page/change_form.html

```diff
@@ -17,6 +17,17 @@
 <script src="{% static "cms/js/dist/bundle.admin.changeform.min.js" %}" type="text/javascript"></script>
 {% endblock %}
 
+{% block content_title %}
+    {% if title %}<h1>{{ title }}</h1>{% endif %}
+    {% if not popup %}
+        <ul class="object-tools">
+            <li>
+                <a href="../{{ page.id }}/{{ language }}/preview/" target="_blank">{% trans "View on site" %}</a>
```

Comment:
I checked the history and cms, there seems to be no url for this case (it was once there like this) @yakky maybe you know?